### PR TITLE
[HIU_000] | Dheeraj/Gayatri | Handle null pointers

### PR DESCRIPTION
**Pull request by dheeraj-p**

### CODE REVIEW

File: src/main/java/in/org/projecteka/hiu/SecurityConfiguration.java [hunk 1]
```diff
@@ -130,21 +128,20 @@ public SecurityContextRepository contextRepository(GatewayTokenVerifier gatewayT
...
+            var token = exchange.getRequest().getHeaders().getFirst(authHeader);
 
-            if (isCMPatientRequest(path, exchange.getRequest().getMethod())) {
-                var patientToken = exchange.getRequest().getHeaders().getFirst(authHeader);
-                return isEmpty(patientToken)
-                       ? error(unauthorizedRequester())
-                       : checkUserToken(patientToken).switchIfEmpty(error(unauthorizedRequester()));
+            if (isEmpty(token)) {
+                return error(unauthorizedRequester());
             }
```
Comment (ganesanarun) [inline src/main/java/in/org/projecteka/hiu/SecurityConfiguration.java:138]: Not correct, you can't generalize this check, that's why I did not do this

**Review by leelanakka — APPROVED** [review]

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -15,8 +15,7 @@ hiu:
     maxPageSize: ${MAX_PAGE_SIZE}
     localStoragePath: ${LOCAL_STORAGE_PATH}
     #Time in minutes
-    consentRequestWaitTime: ${DATA_FLOW_CONSENT_REQUEST_WAIT_TIME}
-    consentArtefactWaitTime: ${DATA_FLOW_CONSENT_ARTEFACT_WAIT_TIME}
+    dataFlowRequestWaitTime: ${DATA_FLOW_REQUEST_WAIT_TIME}
```
Comment (ganesanarun) [inline src/main/resources/application.yml]: Add default fall back value

Comment (dheeraj-p) [reply src/main/resources/application.yml]: Sure.

File: src/main/java/in/org/projecteka/hiu/SecurityConfiguration.java (hunk 1 above)
Comment (dheeraj-p) [reply src/main/java/in/org/projecteka/hiu/SecurityConfiguration.java:138]: Fixed

**Pull request merged**